### PR TITLE
Add a contributing guide

### PR DESCRIPTION
**Pull request by ChaosMTA**
This file will give new pull requests a warning to adhere to the projects guidelines and is a good reminder of our standards.
https://help.github.com/articles/setting-guidelines-for-repository-contributors/

This is related to issue #1264 

Feel free to suggest edits.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  * [Issues Page](https://github.com/GameServerManagers/LinuxGSM/issues) provides a list of areas that could use some work,
+  * [Developing LGSM](https://github.com/GameServerManagers/LinuxGSM/wiki/Developing-LGSM) gives a detailed guide on developing LGSM,
+  * [LGSM Exit Codes](https://github.com/GameServerManagers/LinuxGSM/wiki/LGSM-Exit-Codes) describes and gives an explanation for exit codes,
+  * [gsquery](https://github.com/GameServerManagers/LinuxGSM/wiki/gsquery.py) describes the uses of the gsquery.py file, and
+  * [Branching](https://github.com/GameServerManagers/LinuxGSM/wiki/Branching) is our final guide to submitting changes.
+  
+## Testing
+
+Please make sure all the code you write is working properly **before** you create a pull request. Information on debugging can be found in the following document:
+[Debug Command](https://github.com/GameServerManagers/LinuxGSM/wiki/debug)
```
Comment (UltimateByte) [inline CONTRIBUTING.md]: dev-debug is even more useful for debugging code!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# How to contribute
```
Comment (UltimateByte) [inline CONTRIBUTING.md]: > How to contribute to LinuxGSM

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+I'm really glad you're reading this, because if you are then you have shown an interest in helping make LinuxGSM great.
```
Comment (UltimateByte) [inline CONTRIBUTING.md]: We are*

File: CONTRIBUTING.md (hunk 1 above)
Comment (ChaosMTA) [inline CONTRIBUTING.md]: Are you suggesting removing line 18 then? There is no wiki page on dev-debug except for the page I list directly below.

**Comment by UltimateByte** [comment]
Could also add the fact that people should open an issue when they start working on something, helps avoid having two people working on the same thing and following what's going on with the work.

**Pull request merged**